packages/@storefront/flux-capacitor/test/unit/core/store/middleware.ts

```diff
@@ -427,6 +432,48 @@ suite('Middleware', ({ expect, spy, stub }) => {
     });
   });
 
+  describe('redirectAnalyzer()', () => {
+    let next;
+
+    beforeEach(() => {
+      next = sinon.spy();
```

Comment:
`next` is already assigned as a `spy()` in the top-level `beforeEach`, so this is redundant.

Reply:
Noted, will remove.